lstests/doc/README.md

```diff
@@ -0,0 +1,688 @@
+        default value "127.0.0.1" is supported.
+    port : int
+        Listening port of launcher service.
+        If there are multiple launcher service running in the background,
+        they must have different listening ports.
+        Default is 1234.
+    wdir : str
+        Working directory.
+        Default is the build folder.
+    file : str
+        Path to local launcher service file.
+        Can be either absolute or relative to the working directory.
+    gene : str
+        Path to the genesis file.
+        Can be either absolute or relative to the working directory.
+    start : bool
+        Always start a new launcher service.
+        Note that if to start a new instance alongside the existing ones,
+        make sure the listening ports are different. Otherwise, the
+        new launcher service will issue an error.
+        Default is False (will not start a new one if there is an existing
+        launcher service running in the background).
+    kill : bool
+        Kill existing launcher services (if any).
+        Kill all the existing launcher services running in the background
+        (and freshly start a new one).
+        Default is False (will not kill existing launcher services; instead
+        will connect to an existing launcher service).
+### `Cluster`
+
+A `Cluster` object represents a cluster of nodes running on launcher service. It is the major proxy for tests to communicate with launcher service. A `Cluster` object must have a `Service` object registered to it at initialization.
+
+ After configuration, the node cluster will be launched in one of two modes:
+
+1. BIOS mode, using `eosio.bios` to set producers;
+
+2. regular mode, using `eosio.token` to create and issue tokens, and using
+   `eosio.system` to vote.
+
+By default, a cluster is launched in the BIOS mode.
+
+A detailed explanation of the parameters to initialize a `Cluster` object can be found in its docstring.
+
+```
+Parameters
+----------
+service : Service
+    Launcher service object on which the node cluster will run
+cidr : str
+    Smart contracts directory.
+    Can be either absolute or relative to service's working directory.
+cluster_id : int
+    Cluster ID to launch with.
+    Tests with different cluster IDs can be run in parallel.
+    Valid range is [0, 30). Default is 0.
+node_count : int
+    Number of nodes in the cluster.
+    Default is 4.
+pnode_count : int
+    Number of nodes with at least one producer.
+    Default is 4.
+producer_count : int
+    Number of producers in the cluster.
+    Default is 4.
+unstarted_count : int
+    Number of unstarted nodes.
+    Default is 0.
+topology : str
+    Cluster topology to launch with.
+    Valid choices are "mesh", "bridge", "line", "ring", "star", "tree".
+    Default is "mesh".
+center_node_id : int
+    Center node ID (for bridge or star topology).
+    If topology is bridge, center node ID cannot be 0 or last one.
+    No default value.
+tokens_supply : float
```

Comment:
Since the `eosio.token` contract should not belong in the new integration test framework, this parameter should be removed.

Reply:
Parameter removed. In general, the regular launch mode has been removed from the source code. The doc has been updated accordingly. Thank you.